projects/packages/action-bar/src/action-bar.jsx

```diff
@@ -0,0 +1,32 @@
+import { render } from 'preact';
```

Comment:
@mtias mentioned on the p2 that he would avoid preact unless we can work out an issue with multiple versions of preact being available. I'm not sure about this issue but it sounds like it needs investigation pe7F0s-7b-p2#comment-77